codegen/render/manifests_renderer.go

```diff
@@ -3,27 +3,36 @@ package render
 import (
 	"strings"
 
+	"cuelang.org/go/cue"
+	"cuelang.org/go/encoding/openapi"
+	"cuelang.org/go/encoding/protobuf"
+	eris "github.com/rotisserie/eris"
+	"github.com/solo-io/anyvendor/anyvendor"
+	"github.com/solo-io/go-utils/stringutils"
+	"github.com/solo-io/skv2/codegen/collector"
 	"github.com/solo-io/skv2/codegen/kuberesource"
-
+	"github.com/solo-io/skv2/codegen/model"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/yaml"
 )
 
 // creates a k8s resource for a group
 // this gets turned into a k8s manifest file
-type MakeResourceFunc func(group Group) []metav1.Object
+type MakeResourceFunc func(group Group, oapiSchemas kuberesource.OpenApiSchemas) ([]metav1.Object, error)
```

Comment:
Rather than change this function, should we put the schemas on the `Group`?

Reply:
I'll defer to @ilackarms for how best to represent this

Reply:
i tend to agree with Eitan, i think it's better to move `oapiSchemas` into the place we care about them (inside the CRD constructor function). though it seems we will need to put `protoDir` here

Reply:
would that cause us to run the `collector` an additional time for every group?

Reply:
Moved to a field in `Group`. And no, this would not cause the collector run an additional time per group.